sdk/quantum/Azure.Quantum.Jobs/README.md

```diff
@@ -74,12 +77,22 @@ var quantumJobClient =
         credential);
 ```
 
+### Get Container SAS URI
+
+Create a storage container where to put your data.
```

Comment:
This isn't actually creating a container is it?  I assume it just generates a SAS URI based on the name.